XbSymbolDatabase.c

```diff
@@ -1386,11 +1386,14 @@ bool XbLibraryScan(custom_scan_func_t custom_scan_func,
                         if (!scan_ret) {
                             continue;
                         }
-                        return scan_ret;
                     }
                 }
             }
-            break;
+            // We're not using force break here since we are combining LTCG flag to std flag's oovpa database such as D3D8.
+            // Plus break out of the loop when custom manual scan is done.
+            if (scan_ret) {
```

Comment:
The break could still be enabled, if `LibraryFlag` contains only one bit set.

This could be checked like : ``(LibraryFlag & LibraryFlag-1) == 0)`

Reply:
That is a good idea, though... it would save a few cpu cycles. But I'll go ahead re-update to use your idea design.

Reply:
Hmm... I realize it will not work because `LibraryFlag` will always have one bit flag set. I'll investigate another possible solution.

Reply:
Found an alternative solution. `(SymbolDBList[d2].LibSec.library & ~LibraryFlag) == 0` will check if there are other value sets in the database's library flags.

Plus is confirmed working by do a debug test for each value inspection. And did not break the symbol cache file comparison.